consumer/push_consumer.go

```diff
@@ -493,6 +493,8 @@ func (pc *pushConsumer) pullMessage(request *PullRequest) {
 			goto NEXT
 		}
 
+		pc.processPullResult(request.mq, result, sd)
```

Comment:
the L507 will overwrite result of `pc.processPullResult(request.mq, result, sd)`, could you delete L507 and retest？

Reply:
dome